storagegroup/storagegroup.go

```diff
@@ -140,6 +140,7 @@ func (sg *StorageGroup) SetValidationDataHash(hash checksum.Checksum) {
 // See also SetExpirationEpoch.
 func (sg StorageGroup) ExpirationEpoch() uint64 {
```

Comment:
Let's drop them from SDK while we can. This thing is deprecated for a reason and we have no SGs in the wild.

Reply:
And it's deprecated for more than a year now, btw.

Reply:
Do we want to remove SetExpirationEpoch as well? It is using the deprecated v2.SetExpirationEpoch
@roman-khimov

Reply:
Sure, both can go.

Reply:
Done